src/components/editMeasure/measureDetails/MeasureDetails.tsx

```diff
@@ -1,8 +1,8 @@
-import React from "react";
+import * as React from "react";
```

Comment:
I've seen this import alias added and removed a few times now. Let's make sure we note the preferred style to avoid extraneous changes.

Reply:
Not sure why it is required, it looks like we got it from the merge. Generally, a default import for react is good enough, instead of importing the entire library.